src/tools/ScenarioMeasurement/Startup/Perfcollect.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.Diagnostics.Tracing.Parsers;
+using ScenarioMeasurement;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace ScenarioMeasurement
+{
+    public class Perfcollect : IDisposable
+    {
+        private readonly string filepath = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).FullName, @"Startup/perfcollect");
+        private ProcessHelper perfcollectProcess;
```

Comment:
Can you guarantee all uses of this tool will be in the directory you expect here?

Reply:
Hmm I'm not sure.. Currently only Startup tool downloads and uses it, so the script is in that project folder. Are you suggesting we should move it out so other projects can use it? Or the script should go to wherever the working directory is set to in Startup?

Reply:
hmmm just got what you mean. Maybe the path should be passed in.